lib/ProxySQL_Admin.cpp

```diff
@@ -5338,7 +5456,19 @@ void ProxySQL_Admin::flush_admin_variables___database_to_runtime(SQLite3DB *db,
 		{
 			if (variables.restapi_enabled != variables.restapi_enabled_old) {
 				if (variables.restapi_enabled) {
-					AdminRestApiServer = new ProxySQL_RESTAPI_Server(variables.restapi_port);
+					std::function<std::shared_ptr<httpserver::http_response>(const httpserver::http_request&)> prometheus_callback {
+						[this](const httpserver::http_request& request) {
+							auto headers { request_headers(request) };
+							auto serial_response { this->serial_exposer(headers) };
```

Comment:
We should have something similar also few lines after, after:
```
            } else {
                if (variables.restapi_port != variables.restapi_port_old) {
```

Reply:
Fixed!